modules/about-the-must-gather-tool.adoc

```diff
@@ -0,0 +1,18 @@
+// Module is included in the following assemblies:
```

Comment:
Is there a reason to create this new module rather than reusing the content at https://docs.openshift.com/container-platform/4.13/support/gathering-cluster-data.html#about-must-gather_gathering-cluster-data ?

Also, if this is specific to gitops, it might be good to make the title more reflective of that, e.g. `about-the-must-gather-tool-gitops.adoc`so it doesn't cause confusion for others in the repository.

Reply:
Thank you @abrennan89 for pointing this out. The already published content is almost generic and we can reuse the same module here. So, i reused that module in this PR.